### ISSUE TITLE
App can't launch

### ISSUE THREAD

**Issue by arjanvlek** [saagarjha/OpenAdBlock#6]
Wanted to install this as all App Store ad blockers require iPhone 5s+ / iPad air+ while this one says "Also works on iPhone 5 / 5c", which also means iPad 4 (A6 CPU).

Issue:
App crashes on launch.

Device: 
iPad 4 32GB (iOS 9.2)

Crash Logs:
[OpenAdBlock  03-01-16 17-40.txt](https://github.com/saagarjha/OpenAdblock/files/76664/OpenAdBlock.03-01-16.17-40.txt)
[OpenAdBlock  03-01-16 17-40 2.txt](https://github.com/saagarjha/OpenAdblock/files/76665/OpenAdBlock.03-01-16.17-40.2.txt)

xcode screenshot: 
![schermafbeelding 2016-01-03 om 17 42 18](https://cloud.githubusercontent.com/assets/10106652/12079686/4af49f86-b242-11e5-9fe6-cf9e25145be4.png)

**Comment by jusleg** [comment]
Apple changed it. Now it only works on 64 bit devices. Sorry for the problem.

**Comment by arjanvlek** [comment]
So it crashed because of 64-bit CPU instructions being run on a 32-bit CPU? And there's nothing I can do to get it running?
If so, then feel free to close the issue. If a workaround exists, I'm ready to be informed about it.

**Comment by jusleg** [comment]
I'm afraid there is nothing to do... It worked before but they changed
something and now nothing work on 32 bit devices.

**Comment by saagarjha** [comment]
Content Blockers do not work on 32-bit devices. This is an Apple-imposed restriction; Apple has removed the "Content Blockers" section from Settings. Thus, you should be able to run the app on your 32-bit device (i.e. the crash is with your Xcode/device, not Apple's restrictions), but won't be able to actually turn on the blocking.

**Comment by saagarjha** [comment]
I've found a way to get the content blockers to load; I'll be posting it here once it's a bit more polished.

**Comment by saagarjha** [comment]
@arjanvlek I've updated the repo; could you try it out?

**Comment by arjanvlek** [comment]
@saagarjha Yes, I will try it out this afternoon. ~~Btw, is your fix meant for the app crashing upon launch, or to get content blockers to load on 32-bit iOS devices?~~
Nvm, just saw it is indeed for the 32 bits devices. Will try it on my iPad 4 (iOS 10) this afternoon :)

**Comment by hontz1** [comment]
Hallo, 
the app loads, but hangs in a white screen on iPhone 5c.
Maybe because i updated the coockielist, or there were errors within Pods-OpenAdBlock-frameworks.sh, i simply deleted.
Don't mess your time with old 32-bit iOS cookie blocker. Please can someone tell me why my 5 years old working iPhone should be thrown away? It is working fine. crazy information-technology lifestyle product
____________________________________WRONG______________________________________________________________________>
In **Pods-OpenAdBlock-frameworks.sh**, i deleted, because of error: _`unsupported option`_

```
set -o pipefail

function on_error {
echo "$(realpath -mq "${0}"):$1: error: Unexpected failure"
}
trap 'on_error $LINENO' ERR

```
and in same file, i deleted because of an other error: _`unexpected syntax {`_

```
RSYNC_PROTECT_TMP_FILES=(--filter "P .*.??????")
```
<__________________________________________________________________________________________________________WRONG
Here is my self-generated **blockerList.json** (I wanted eu-cookie-question-popup blocking only).:
[blockerList.json.zip](https://github.com/saagarjha/OpenAdBlock/files/5986566/blockerList.json.zip)

Maybe someone could help? Why did they deleted all the 32-bit app versions from app-store?
I bet, 90 % of the worldwide machine-users don't really need new hardware since 10 years.

> No!, they need!

> 
Thank you, good bye
Peter

**Comment by hontz1** [comment]
UPDATE:
the app loads now but still produces error.
in **Pods-OpenAdBlock-frameworks.sh** i changed `sh` to `bash`.
After app is loading i get:
error: ```Error Domain=SFErrorDomain Code=1 "(null)"```
and i don't see safari-setting
maybe it is an Xcode 8+10 issue. Xcode 12 can't handle this swift version.

Could someone please help?
Thank you. stay healthy.
![108325214-24851a00-71c9-11eb-84ae-7e5a4ef93c21](https://user-images.githubusercontent.com/51762805/108401575-c5e98b80-721c-11eb-89d1-9b7a8f729492.PNG) ![108325603-834a9380-71c9-11eb-83c1-c24e5259690f](https://user-images.githubusercontent.com/51762805/108401589-c8e47c00-721c-11eb-9674-2b8e46d80ac3.PNG) ![108325222-27800a80-71c9-11eb-829e-9bf0d067f205](https://user-images.githubusercontent.com/51762805/108401606-ceda5d00-721c-11eb-9fa8-f5bb6f947e7d.PNG) try blockparty build 2016 sucessful instead
